droom-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
You should consider using a formatter like Prettier to make your code more readable.